### PR TITLE
Verbose make

### PR DESCRIPTION
**Pull request by Blackhex**
Makes make output verbose and allows overriding the `BUILD_MAKE_OPTIONS` from outside of scripts.

### CODE REVIEW

File: .github/scripts/build-binutils.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ BINUTILS_VERSION=${BINUTILS_VERSION:-binutils-master}
 
 TARGET=${TARGET:-aarch64-w64-mingw32}
 BUILD_PATH=${BUILD_PATH:-$PWD/build-$TARGET}
-BUILD_MAKE_OPTIONS=-j$(nproc)
+BUILD_MAKE_OPTIONS=${BUILD_MAKE_OPTIONS:V=1 -j$(nproc)}
```
Comment (eukarpov) [inline .github/scripts/build-binutils.sh:7]: What about moving these almost the same settings out? Less changes for adjustment, less chances to forget something.

Comment (Blackhex) [reply .github/scripts/build-binutils.sh:7]: The default value here serves only the purpose when the script is called alone, e.g. locally.

Comment (eukarpov) [reply .github/scripts/build-binutils.sh:7]: for instance to move BUILD_MAKE_OPTIONS to a separate bash script
include the script everywhere where it is needed.
Overwriting default value will be working the same way, however default value will be placed only in one place.

Comment (Blackhex) [reply .github/scripts/build-binutils.sh:7]: OK, let me try whether this approach will be feasible. Thank you for the suggestion.

Comment (Blackhex) [reply .github/scripts/build-binutils.sh:7]: The feedback adressed in the https://github.com/ZacWalk/mingw-woarm64-build/pull/49. This PR will be updated to contain only make options change once that one will be merged.

**Review by eukarpov — APPROVED** [review]

**Pull request merged**